Defs/Ammo/HighCaliber/35x228mmNATO.xml

```diff
@@ -0,0 +1,301 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Defs>
+
+	<ThingCategoryDef>
+		<defName>Ammo35x228mmNATO</defName>
+		<label>35x228mm NATO</label>
+		<parent>AmmoHighCaliber</parent>
+		<iconPath>UI/Icons/ThingCategories/CaliberHighCaliber</iconPath>
+	</ThingCategoryDef>
+
+	<!-- ==================== AmmoSet ========================== -->
+
+	<CombatExtended.AmmoSetDef>
+		<defName>AmmoSet_35x228mm</defName>
+		<label>35x228mm NATO</label>
+		<ammoTypes>
+			<Ammo_35x228mmNATO_AP>Bullet_35x228mmNATO_AP</Ammo_35x228mmNATO_AP>
+			<Ammo_35x228mmNATO_Incendiary>Bullet_35x228mmNATO_Incendiary</Ammo_35x228mmNATO_Incendiary>
+			<Ammo_35x228mmNATO_HE>Bullet_35x228mmNATO_HE</Ammo_35x228mmNATO_HE>
+			<Ammo_35x228mmNATO_Sabot>Bullet_35x228mmNATO_Sabot</Ammo_35x228mmNATO_Sabot>
+		</ammoTypes>
+		<similarTo>AmmoSet_Autocannon</similarTo>
+	</CombatExtended.AmmoSetDef>
+
+	<!-- ==================== Ammo ========================== -->
+
+	<ThingDef Class="CombatExtended.AmmoDef" Name="Ammo35x228mmNATOBase" ParentName="MediumAmmoBase" Abstract="True">
```

Comment:
[Source](https://ordtech-industries.com/wp-content/uploads/2023/09/OMI-35X228mm.pdf) for some ammo data (putting it here just for tracking purposes)